doc/optimizations.txt

```diff
@@ -260,6 +263,12 @@ Optimization                                              FAST_RUN  FAST_COMPILE
         Taking the log of 0 would generate -inf that will probably generate NaN later.
         We return a closer answer.
 
+    o2
+        FAST_RUN but slow opt like inplace or elemwise_inplace (just excluding inplace for now)
+
+    o3
```

Comment:
you swapper the meaning of o3 and o2. o3 is fast_run minus stuff, o2 is fast_compile + stuff.